src/main/java/org/jabref/gui/fieldeditors/AbstractEditorViewModel.java

```diff
@@ -65,8 +65,7 @@ public void bindToEntry(BibEntry entry) {
                 fieldBinding,
                 newValue -> {
                     if (newValue != null) {
-                        // Controlsfx uses hardcoded \n for multiline fields, but JabRef stores them in OS Newlines format
-                        String oldValue = entry.getField(field).map(value -> value.replace(OS.NEWLINE, "\n").trim()).orElse(null);
+                        String oldValue = entry.getField(field).map(String::trim).orElse("");
```

Comment:
The source field always uses \n as separator internally. We need this check to prevent the change detection

Reply:
> The source field always uses \n as separator internally.

I saw it in the debugger.

Therefore

```java
value.replace(OS.NEWLINE, "\n")
```

is a no-op.


Reason: 

Let's split up the cases.

- OS.NEWLINE is `\r\n`. Since "**always uses \n **" is true, `\r\n` is never matched. So no replacement is made
- OS.NEWLINE is `\n`.  Since "**always uses \n **" is true, `\n` is replaced by `\n`. It is replaced by **the same** character. The identity. Thus, no change.

Did I miss something?

Side note: During the file write, the newlines are now normalized. The whole file does have the same newline separator. It came in with https://github.com/JabRef/jabref/pull/8238.

Reply:
Yes, we have to compare old entry and new Entry. New entry is what controlsfx uses.
Old Entry however is what comes from the stored bib file, might have had OS.Newline  -> Replaced with \n and compare with new entry so that we don't create a diff "database changed"

Reply:
Yes, but that does not unify OS.newline 

1. If I have a bib file with OS.newline chars e.-g \r\n  and now I change the bibtex source -> This will trigger a change

Reply:
OMG. Thus, if you open the attached file (CRLF terminators), go into the comment field, the cursor will jump?

[crlf.bib.txt](https://github.com/JabRef/jabref/files/12658430/crlf.bib.txt)

I now hard-coded CRLF in the code (instead of relying on OS.Newline)

Reply:
That's why we replace the chars.